### PR TITLE
add intbitset python package

### PR DESCRIPTION
**Pull request by gtonkinhill**
Hi @conda-forge/help-python-c

Please review this a recipe for the python package intbitset. 

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/intbitset/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  imports:
+    - intbitset
+
+about:
+  home: http://github.com/inveniosoftware/intbitset/
+  license: GNU Lesser General Public v3 or later (LGPLv3+)
+  license_family: LGPL
+  license_file: lgpl-3.0.txt
+  summary: C-based extension implementing fast integer bit sets.
+  doc_url:
```
Comment (xhochy) [inline recipes/intbitset/meta.yaml:37]: Please remove empty sections.

File: recipes/intbitset/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - python
+    - six
+
+test:
+  imports:
+    - intbitset
+
+about:
+  home: http://github.com/inveniosoftware/intbitset/
+  license: GNU Lesser General Public v3 or later (LGPLv3+)
```
Comment (xhochy) [inline recipes/intbitset/meta.yaml:33]: Please use the SPDX classifier 
```suggestion
  license: LGPL-3.0-or-later
```

File: recipes/intbitset/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - six
```
Comment (xhochy) [inline recipes/intbitset/meta.yaml:22]: Is `six` needed for build? Looks like not.

File: recipes/intbitset/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - intbitset
+
+about:
+  home: http://github.com/inveniosoftware/intbitset/
+  license: GNU Lesser General Public v3 or later (LGPLv3+)
+  license_family: LGPL
+  license_file: lgpl-3.0.txt
```
Comment (xhochy) [inline recipes/intbitset/meta.yaml:35]: Looking at the upstream code, you could put `LICENSE` here and it should work, no need for bundling it with the recipe.

File: recipes/intbitset/meta.yaml (hunk 3 above)
Comment (gtonkinhill) [reply recipes/intbitset/meta.yaml:22]: I think six is needed so I've added it back in

Comment (xhochy) [reply recipes/intbitset/meta.yaml:22]: Only remove it in `host`, not in `run`. It is required to run the package but not to build it.

File: recipes/intbitset/meta.yaml (hunk 4 above)
Comment (gtonkinhill) [reply recipes/intbitset/meta.yaml]: I was not sure exactly what you meant here, but I've removed the license file and replaced the section with 'LICENSE'

**Comment by gtonkinhill** [comment]
Hi @conda-forge/help-python-c

Please could someone review the recent fixes for this recipe.

**Comment by xhochy** [comment]
See https://github.com/conda-forge/staged-recipes/pull/11233/commits/df7a58f3e057cbfc5e378660be0db95a3744addb#r404635180

**Comment by xhochy** [comment]
Thanks @gtonkinhill !

**Comment by gtonkinhill** [comment]
Thanks for the review @xhochy!

**Pull request merged**